app/workers/kyc/kycaid/verifications_worker.rb

```diff
@@ -0,0 +1,27 @@
+require 'sidekiq'
+module KYC
+  module Kycaid
+    class VerificationsWorker
+      include Sidekiq::Worker
+
+      def perform(params)
+        params = params.symbolize_keys
+        params.slice(:verification_id, :applicant_id, :verified, :verifications)
+
+        user = Profile.find_by(applicant_id: params[:applicant_id]).user
+        verification = ::KYCAID::Verification.fetch(params[:verification_id])
```

Comment:
fetch a verification. This way we will be sure info is really received from KYCAID